parallel make fails: `ld: cannot find -lnfcauth: No such file or directory`

On my 16-core Threadripper machine when running `make -j32` I see the following build error:
```
libtool: link: x86_64-pc-linux-gnu-gcc -shared  -fPIC -DPIC  .libs/nfcauth.o .libs/nfcauth_backend_config.o   -lnfc -lusb -Wl,--as-needed -lcrypt -ldl  -march=native -mtune=native -O2 -Wl,-O1 -Wl,--hash-style=gnu -Wl,--sort-common -Wl,-z -Wl,relro -Wl,-z -Wl,now   -Wl,-soname -Wl,libnfcauth.so.0 -o .libs/libnfcauth.so.0.0.0
/usr/lib/gcc/x86_64-pc-linux-gnu/13/../../../../x86_64-pc-linux-gnu/bin/ld: cannot find -lnfcauth: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:561: pam_nfc.la] Error 1
make: *** Waiting for unfinished jobs....
```
Even `make -j2` reveals this error. Only when I fall back to serial build (`make -j1`), build succeeds.

--- a/Makefile.am
+++ b/Makefile.am
@@ -35,3 +35,5 @@ pam_nfc_add_LDADD = \
 	libnfcauth.la
 
 EXTRA_DIST = nfcauth.h
+
+pam_nfc.la: libnfcauth.la

